components/mpas-seaice/docs/user-guide/index.md

```diff
@@ -0,0 +1,64 @@
+# User's Guide
+
+Guidance for using E3SM is available from [E3SM's public web site](<https://e3sm.org/model/running-e3sm/e3sm-quick-start>).
+
+## Configuring MPAS-seaice
+
+MPAS-seaice is controlled using namelist options.
+
+- Default namelist values are found in
+``E3SM/components/mpas-seaice/bld/namelist_files/namelist_defaults_mpassi.xml``.
+- Namelist options are defined in
+``E3SM/components/mpas-seaice/bld/namelist_files/namelist_definitions_mpassi.xml``,
+including type, category (``seaice_model``), group, valid values and a brief description. Each namelist variable is defined in an ``entry`` element.  The content of the element is the documentation of how the variable is used.  Other aspects of the variable's definition are expressed as attributes of the ``entry`` element.
+- Some namelist values or combinations are not allowed and will generate warnings and often abort the code.  The consistency checks for using MPAS-seaice within E3SM are in ``mpas_seaice_initialize`` (subroutines ``seaice_check_configs_coupled``, ``seaice_check_constants_coupled``), and those specific to Icepack can be found in subroutine ``check_column_package_configs`` in ``mpas_seaice_icepack.F``.
```

Comment:
- Users can change namelist options from defaults by entering `<namelist option>` = `<changed value>` as separate lines in the `user_nl_mpassi` file in the case directory.